### DISCUSSION TITLE
Set default Page

### DISCUSSION THREAD

**Discussion by etsde** [outline/outline#6835] [General]
Hello,
is it somehow possible to set a specific page as default page?
For instance, if your domain is outline.example.com, it should show a dedicated default page. In this case, the login section should switch to e.g. outline.example.com/login or so.
Thank you in advance!

**Comment by tommoor** [comment]
This is possible on cloud hosted if you contact support

**Reply by etsde** [reply]
Hey, thanks for the answer. Is there any kind of method (like a Var) for the self-hosted way to do so?

**Reply by tommoor** [reply]
The logic doesn't exist in the community edition I'm afraid

**Reply by etsde** [reply]
Oh boi, can we add this one as feature request, maybe?

**Reply by tommoor** [reply]
It will likely make sense as part of the #1160 functionality
